dgl_ptm/tests/test_model.py

```diff
@@ -76,6 +77,36 @@ def test_data_collection_timestep1(self, model):
         edge_data = xr.open_zarr('my_model/edge_data/1.zarr')
         assert 'weight' in edge_data.variables
 
+    def test_data_collection_period(self, model):
+        if Path('my_model/edge_data/').exists():
+            shutil.rmtree('my_model/edge_data/')
+
+        model.steering_parameters['data_collection_period'] = 3
+
+        model.run()
+
+        assert model.step_count == 5
+        assert Path('my_model/agent_data.zarr').exists()
+        assert not Path('my_model/edge_data/2.zarr').exists()
+        assert Path('my_model/edge_data/3.zarr').exists()
+
+    def test_data_collection_list(self, model):
```

Comment:
The new functionality `data_collection_intervals` is a part of the script `step.py`. But the tests are added to the class `TestDataCollection`. This class is meant to test the functionalities of the script `data_collection.py`. Please move the test to the class `TestPtmStep`. Also, please replace `data_collection_period` and `data_collection_list` with the suggestion i.e. `data_collection_intervals`.